lib/pbench/agent/tool_meister.py

```diff
@@ -72,6 +73,85 @@ class ToolException(Exception):
     pass
 
 
+class PermaTool:
+    def __init__(self, name, tool_opts, logger):
+        self.name = name
+        self.tool_opts = tool_opts.split(" ")
+        self.logger = logger
+        self.install_path = None
+        for opt in self.tool_opts:
+            if opt.startswith("--inst="):
+                if opt[len(opt) - 1] == "\n":
+                    self.install_path = opt[7 : len(opt) - 1]
+                else:
+                    self.install_path = opt[7:]
+                self.logger.debug("FOUND")
```

Comment:
I get that you're detecting and then removing the `--inst=` prefix here, but the code would be easier to read with an explanation, maybe in a comment... although this is where real parameter parsing is a boon. ;-)